Utilities/bootstrap

```diff
@@ -397,22 +397,21 @@ def install_swiftpm(prefix, args):
     runtime_lib_dest = os.path.join(prefix, "lib", "swift", "pm")
     runtime_lib_src = os.path.join(args.bootstrap_dir, "pm")
```

Comment:
Note that for now we’re still installing the bootstrap-built PackageDescription.  This will change in the follow-on PR.